CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.8.0 (Next)
 
+* [#180](https://github.com/mongoid/mongoid-history/pull/180): [#177] Removed deprecation notice - [@sivagollapalli](https://github.com/sivagollapalli).
```

Comment:
The #177 here is messing things up here I believe. It doesn't automatically link anyway, if you want to mention it make it `[#180](...), [#177](...): ...`